app/src/main/java/it/niedermann/owncloud/notes/android/fragment/NotePreviewFragment.java

```diff
@@ -113,6 +139,15 @@ public void onNext(CharSequence charSequence) {
         }
     }
 
+    private Set<String> getExistingNoteRemoteIds() {
+        List<DBNote> notes = db.getNotes(note.getAccountId());
+        Set<String> noteRemoteIds = new HashSet<>();
+        for (DBNote note : notes) {
+            noteRemoteIds.add(String.valueOf(note.getRemoteId()));
+        }
+        return noteRemoteIds;
+    }
+
```

Comment:
We should move this into NoteSQLiteOpenHelper class as a direct SQL-query for performance reasons.

Reply:
alright, will do

Reply:
please review again and resolve if you are happy with the changes.

Ah and thanks for the review in general. Much appreciated :)